README update (#687)
discordrb/discordrb: README.md

See also: [Documentation](https://www.rubydoc.info/gems/discordrb), [Tutorials](https://github.com/discordrb/discordrb/wiki)

## Introduction

`discordrb` aims to meet the following design goals:

1. Full coverage of the public bot API.
2. Expressive, high level abstractions for rapid development of common applications.

> 3. Friendly to Ruby beginners, as well as beginners of open source contribution.

Reviewer: ```suggestion
3. Friendly to Ruby beginners and beginners of open source contribution.
```

If you enjoy using the library, consider getting involved with the community to help us improve and meet these goals!

**You should consider using `discordrb` if:**

> - You need a bot - and fast - for small or medium sized communities, and don't want to be bogged down with "low level" details. Getting started takes minutes, and with utilities like a command parser and tools for modularization make it simple to quickly add or change your bots functionality.

Reviewer: ```suggestion
- You need a bot - and fast - for small or medium sized communities, and don't want to be bogged down with "low level" details. Getting started takes minutes, and utilities like a command parser and tools for modularization make it simple to quickly add or change your bots functionality.
```

> - You like or want to learn Ruby, or want to contribute to a Ruby project. A lot of our users are newcomers to Ruby, who eventually make their first open source contributions with us. We have an active Discord channel with experienced members who will happily help  you get involved, either as a user or contributor.

Reviewer: ```suggestion
- You like or want to learn Ruby, or want to contribute to a Ruby project. A lot of our users are new to Ruby, and eventually make their first open source contributions with us. We have an active Discord channel with experienced members who will happily help you get involved, either as a user or contributor.
```
